### ISSUE TITLE
What's the difference between `folder_id` and `parent_folder`?

### ISSUE THREAD

**Issue by lpezet** [terraform-google-modules/terraform-google-bootstrap#300]
### TL;DR

As per the `variables.tf` file:
```
variable "parent_folder" {
  description = "GCP parent folder ID in the form folders/{id}"
  default     = ""
  type        = string
}
...
variable "folder_id" {
  description = "The ID of a folder to host this project"
  type        = string
  default     = ""
}
```

### Additional information

Looking at [terraform-example-foundation](https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/0-bootstrap/main.tf#L47) it look like it's an optional folder where to host the [seed-project](https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/master/main.tf#L65).
But then every `google_folder_iam_member` is based on `parent_id` (if `parent_folder` provided).
So is it possible to have a `folder_id` OUTSIDE the `parent_folder`?
In the case of `terraform-example-foundation`  they create a subfolder within `parent_folder`, which makes sense.
Just confusing is all...
